### PR TITLE
community/claws-mail: remove openssl dependency

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: community/claws-mail/APKBUILD [hunk 1]
```diff
@@ -4,7 +4,7 @@ pkgname=claws-mail
 pkgver=3.17.1
 pkgrel=1
```
Comment (andypost) [inline community/claws-mail/APKBUILD]: it needs bump to rebuild

Comment (J0WI) [reply community/claws-mail/APKBUILD]: Oops, fixed

**Comment by ncopa** [comment]
I don't think claws-mail uses either openssl or libressl. I think libressl-dev can simply be removed from makedepends.

**Comment by J0WI** [comment]
I removed the dependency completely.

**Comment by J0WI** [comment]
The LibreSSL dependency has been replaced in 7c1ac73c4b4001bd38ada2882d00ec9ba899254b, so I rebased this.

**Comment by andypost** [comment]
Commited 5fc8f5be

**Pull request closed without merging**